docquality.rst

```diff
@@ -76,9 +80,10 @@ Helping with the Developer's Guide
 
 .. highlight:: console
 
-The Developer's Guide uses the same process as the main Python documentation,
-except for some small differences.  The source lives in a `separate
-repository`_ and bug reports should be submitted to the `the GitHub tracker`_.
+The Developer's Guide (what you're reading now) uses the same process as the
+main Python documentation, except for some small differences.  The source
+lives in a `separate repository`_ and bug reports should be submitted to
+the `its GitHub tracker`_.
```

Comment:
One small correction still needed after the PR split ~~the~~. Thanks @csabella 👍